.github/workflows/push-to-ghcr.yml

```diff
@@ -0,0 +1,35 @@
+name: Push package to GHCR
+
+env:
+  DOTNET_VERSION: '6.0.x'
+
+on:
+  push:
+
+jobs:
+      push-store-image:
+        runs-on: ubuntu-latest
+        steps:
+          - name: 'Checkout GitHub Actions'
+            uses: actions/checkout@main
+
+          - name: Import GPG key
+            uses: crazy-max/ghaction-import-gpg@v6
+            with:
+              gpg_private_key: ${{ secrets.HELM_SIGN_PGP_PRIVATE_KEY }}
+              passphrase: ''
+
+          - name: Export PGP Private Key
+            run: |
+              gpg --export-secret-keys 'Helm Package Signing' > ~/.gnupg/private.gpg
```

Comment:
Do you need to export the key if it's already in the keyring? Like if you remove this and the `--keyring ~/.gnupg/private.gpg` flag from the helm command.

Reply:
Yeah, sadly need to ensure it is the right key format to make it work! This export make sure it is.